test/test_misc.py

```diff
@@ -398,13 +399,15 @@ def test_elasticity_new():
     stokes_knl = StokesletKernel(3, 0, 1, "mu1", 0.5)
     stokes_knl2 = ElasticityKernel(3, 0, 1, "mu1", 0.5)
     elasticity_knl = ElasticityKernel(3, 0, 1, "mu1", "nu")
+    elasticity_helper_knl = LineOfCompressionKernel(3, 0, "mu1", "nu")
```

Comment:
This should be part of `test_pde_check_kernels`, too. Basic elasticity and Stokes are missing, too, but that's unrelated to this PR. (hence #54)

Reply:
Fixed in 6ef2fad2bc4d91d051eb48145b59f3ff4501ed47